inject.js

```diff
@@ -448,6 +501,13 @@
       });
   }
 
+  function setSpeed(controller,video,speed) {
```

Comment:
Moved setSpeed into its own function in order to not rely on ratechange listener to update the controller.